test/unit/test_utils.py

```diff
@@ -29,12 +30,30 @@
 pytestmark = [pytest.mark.unit, pytest.mark.local]
 
 
+@pytest.yield_fixture
+def patch_ec(mocker):
+    mocker.patch.object(aws_encryption_sdk.internal.crypto.authentication, "ec")
+    yield aws_encryption_sdk.internal.crypto.authentication.ec
+
+
 def test_prep_stream_data_passthrough():
     test = aws_encryption_sdk.internal.utils.prep_stream_data(io.BytesIO(b"some data"))
 
     assert_prepped_stream_identity(test, io.BytesIO)
 
 
+def test_verify_interface(patch_ec):
```

Comment:
Suggestion: Rename `test_verify_ec_interface`
```suggestion
def test_verify_ec_interface(patch_ec):
```